src/pybel/io/graphml.py

```diff
@@ -60,7 +60,7 @@ def _to_graphml_simple(graph: BELGraph) -> nx.MultiDiGraph:
 
 
 def _to_graphml_umbrella(graph: BELGraph) -> nx.MultiDiGraph:
-    """Convert a BEL graph to GraphML XML file by previously canonicalizing the nodes.
+    """Convert a BEL graph to GraphML XML file by using :mod:`pybel.canonicalize` in the nodes.
```

Comment:
Sorry I think there's still a misunderstanding - I want you to stop using this kind of terminology. It doesn't tell the user anything because the word canonicalize is totally meaningless outside of the very specific way that I (as the developer of the library) used it

Reply:
It should be fine now.